.mailmap

```diff
@@ -37,6 +37,8 @@ Artem Zaytsev <a.arepo@gmail.com>
 Artur G Vieira <vieira.artur.g@gmail.com> Artur Vieira <vieira.artur.g@gmail.com>
 Arnout Kazemier <info@3rd-Eden.com> <3rd-Eden@users.noreply.github.com>
 Asaf David <asafdav2@gmail.com> asafdav2 <asafdav2@gmail.com>
+Ash Cripps <acripps@redhat.com> <ashley.cripps@ibm.com>
+Ash Cripps <acripps@redhat.com> <email@ashleycripps.co.uk>
```

Comment:
You may need to also map `Ashley.Cripps@ibm.com` (as you were appearing in AUTHORS). See https://github.com/nodejs/node/pull/34694#pullrequestreview-466902551 where you were being added again with `ashley.cripps@ibm.com` when the script was run to update AUTHORS.

Reply:
added.